examples/nodejs/lambda-examples/topics-microservice/bin/topics-microservice.ts

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
```

Comment:
can we please add CI for this? It goes in here: 

https://github.com/momentohq/client-sdk-javascript/blob/main/.github/workflows/build.yml#L97-L115

and here:

https://github.com/momentohq/client-sdk-javascript/blob/main/.github/dependabot.yml#L29-L34

Reply:
I don't think i know how to do this. Where am I adding this and how do I test it? Is there a doc somewhere I can read on how we do this overall?

Reply:
so for the first link above - just copy and paste one of the existing blocks. it will just do a `pushd` to go into your examples dir, and then let it run any scripts that you think it should run for validation. My recommendation would be:

```
npm ci
npm run build
npm run cdk synth
```

Then those commands will get automatically run on every PR to make sure nothing got broken.

For the second link, the dependabot one, just copy one of the existing ones and point it to your directory. this will tell github to automatically check to make sure that your code still works with new releases of the momento SDK.

Reply:
added